backend/typescript/rest/camperRoutes.ts

```diff
@@ -1,13 +1,19 @@
 import { Router } from "express";
 
-import { createCamperDtoValidator } from "../middlewares/validators/camperValidators";
+import { isAuthorizedByRole } from "../middlewares/auth";
+import {
+  createCamperDtoValidator,
+  updateCamperDtoValidator,
+} from "../middlewares/validators/camperValidators";
 import CamperService from "../services/implementations/camperService";
 import ICamperService from "../services/interfaces/camperService";
 import { getErrorMessage } from "../utilities/errorUtils";
 import { sendResponseByMimeType } from "../utilities/responseUtil";
 import { CamperDTO } from "../types";
 
 const camperRouter: Router = Router();
+camperRouter.use(isAuthorizedByRole(new Set(["Admin"])));
```

Comment:
remove this line - this check should just be for your edit method, other apis in the camper service like the registration api should be accessible by everyone. you can move `isAuthorizedByRole(new Set(["Admin"]))` to the route definition below